qemu/tests/boot_with_machine_types.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    boot with different machine type:
+    1) get supported machine type
+    2) boot guest with different machine types
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Get supported machine type", logging.info)
+    qemu_binary = utils_misc.get_qemu_binary(params)
+    supported_machine_types, supported_system_name = utils_misc.get_support_machine_type(qemu_binary)
+    machine_types = []
+    machine_type_mapping = {"pc": ["i440FX", "RHEL 6"], "q35": ["Q35"], "pseries": ["pSeries"], "arm64-pci:virt": ["ARM"], "arm64-mmio:virt": ["ARM"], "s390-ccw-virtio": ["S390"]}
```

Comment:
@yanan-fu, i've added the "arm64-mmio:virt" : ["ARM"]  and "s390-ccw-virtio": ["S390"] in my code, but it just displays part of them here. I don't know why, maybe it's too long? I'll find the root cause and update it later, thanks for your reviewing.